packages/bun-types/bun.d.ts

```diff
@@ -3481,6 +3481,13 @@ declare module "bun" {
      * Filtered data consists mostly of small values with a somewhat random distribution.
      */
     strategy?: number;
+
+    library?: "zlib";
```

Comment:
is `"libdeflate"` the other option here? we should add a comment if so

Reply:
typescript autocomplete shows that